### DISCUSSION TITLE
spaCy en_core_web model commercial license

### DISCUSSION THREAD

**Discussion by Desikph** [explosion/spaCy#13698] [Help: Model Advice]
In the spaCy literature, it mentioned that all en_core_web_lg, en_core_web_lg, en_core_web_sm model series trained on OntoNotes -5, but Onto Notes requires Licensing agreement to use the data. In view of this spaCy is open source but these en_core_web model series are not open source and can't be used commercial service products. This violates regulation "cannot use models that are built using non-commercial data sets". Please clarify.

**Comment by honnibal** [comment]
I'm not sure exactly what you've read.

We purchased a commercial license to the OntoNotes data from the LDC, which allows us to train models from it that we can license permissively. What we can't do is distribute the OntoNotes data itself, or distribute an artifact that allows reliable recovery of that data.

We therefore distribute the `en_core_web_*` pipelines under an MIT license. There are some other models which we distribute where we have not been able to obtain a commercial-friendly license to the source data. We distribute those models under different licensing, according to what we're allowed to do.

**Reply by Desikph** [reply]
Thanks for quick response on the licensing part of the models, it helps us in using spaCy models in the services.
